pkg/gameserversets/gameserversets.go

```diff
@@ -120,14 +120,35 @@ func sortGameServersByLeastFullNodes(list []*agonesv1.GameServer, count map[stri
 	return list
 }
 
-// sortGameServersByNewFirst sorts by newest gameservers first, and returns them
-func sortGameServersByNewFirst(list []*agonesv1.GameServer) []*agonesv1.GameServer {
+// sortGameServersByNewFirst sorts by newest gameservers first.
+// If FeatureCountsAndLists is enabled, sort by Priority first, then tie break with newest gameservers.
+func sortGameServersByNewFirst(list []*agonesv1.GameServer, priorities []agonesv1.Priority) []*agonesv1.GameServer {
```

Comment:
Should we change the func name to something like `sortGameServersByPriorities`?

Reply:
In general I dislike the naming of strategy Distributed, since it's not actually distributed. That being said, it would probably make sense to have naming along the lines of `sortGameServersByPackedStrategy` and `sortGameServersByDistributedStrategy`. WDYT?

Reply:
I like it!